main.css

```diff
@@ -0,0 +1,575 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: white;
+  font-family: 'Lato', sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  overflow-x: hidden;
+}
+
+.hide {
+  display: none;
+}
+
+@media screen and (min-width: 375px) and (max-width: 767px) {
+  header {
+    width: 100%;
+    height: 100%;
+    padding: 15px;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .mobile-menu i {
+    margin-bottom: 60px;
+  }
+
+  .menu {
+    background-color: #ec5242;
+    color: #fff;
+    padding-top: 100px;
+    font-size: 30px;
+    position: absolute;
+    width: 100%;
+    height: 800px;
+    mix-blend-mode: multiply;
+    top: -2%;
+    left: -2%;
+    right: -2%;
+    backdrop-filter: blur(15px);
+  }
+
+  .menu ul {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-end;
+    text-align: left;
+  }
+
+  .menu ul a {
+    text-align: left;
+    color: white;
+  }
+
+  .menuBtn {
+    width: 20px;
+  }
+
+  .close-menu {
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: 30px;
+    color: #fff;
+    z-index: inherit;
+  }
+
+  body::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0.1;
+    z-index: -1;
+    background-image: url('./img/back-img1.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .mobile-menu {
+    display: flex;
+    margin-top: 10px;
+  }
+
+  .desk_nav {
+    display: none;
+  }
+
+  .description {
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    margin-left: 50px;
+  }
+
+  .desc-header {
+    font-size: 25px;
+    color: #f00;
```

Comment:
- Great Job for fixing the previous reviewer's suggestions,👌 I think you missed one thing from the previous reviewer which is updating colors based on the original design.
* kindly use thoase color swatch: #272A31, #EC5242, #D3D3D3 to stick the given design

![Uploading 111.JPG…](https://user-images.githubusercontent.com/61408860/234608705-047bb1f7-d7b4-4c13-baf8-d4f619edbec4.png)